file/1.5/modules/ROOT/pages/file-studio-configuration.adoc

```diff
@@ -0,0 +1,120 @@
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add File Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `file` in the search field.
+. Click *File Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with File Connector:
+
+* *On New or Updated File* +
+Initiates a flow by polling a directory for files that have been created or updated and generates a message for each file found.
+* *HTTP Listener* +
+Initiates a flow each time it receives a request on the configured host and port.
+* *Scheduler* +
+Initiates a flow when a time-based condition is met.
+
+For example, to configure the On New or Updated File source, follow these steps:
+
+. In the *Mule Palette* view, select *On New or Updated File*.
+. Drag *On New or Updated File* to the Studio canvas.
+. On the *On New or Updated File* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of *On New or Updated File* in the app.
+. On the *General* tab, specify the connection information for the connector.
+. Set *Host* to specify the FTP server host, for example, `localhost` or `192.168.0.1`.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. On the *On New or Updated File* configuration screen, specify a *Scheduling Strategy* value and optionally complete other fields.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for File Connector, follow these steps:
+
+. In the *Mule Palette* view, select *File* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the source.
```

Comment:
We changed this in the template to remove "right":

Drag the operation onto the Studio canvas, next to the input source. (But we should say "source" instead of "input source" I think.)